packages/ethash/examples/rawExample.ts

```diff
@@ -1,7 +1,7 @@
 import Ethash from '../src'
-const level = require('level-mem')
+import { MemoryLevel } from 'memory-level'
 
-const cacheDB = level()
+const cacheDB = new MemoryLevel()
```

Comment:
Can you cast `cacheDB` to any here if `memory-level` doesn't provide the exact interface conforming to `abstract-level`?